test/demo/filtering-plugin/plugin.js

```diff
@@ -0,0 +1,353 @@
+/*
+ * Modified and maintained by the OpenSeadragon Community.
+ *
+ * This software was orignally developed at the National Institute of Standards and
+ * Technology by employees of the Federal Government. NIST assumes
+ * no responsibility whatsoever for its use by other parties, and makes no
+ * guarantees, expressed or implied, about its quality, reliability, or
+ * any other characteristic.
+ * @author Antoine Vandecreme <antoine.vandecreme@nist.gov>
+ */
+
+(function() {
+
+    'use strict';
+
+    const $ = window.OpenSeadragon;
+    if (!$) {
+        throw new Error('OpenSeadragon is missing.');
+    }
+    // Requires OpenSeadragon >=2.1
+    if (!$.version || $.version.major < 2 ||
```

Comment:
I assume this new version of the plugin actually requires the latest API, right?

Reply:
Yes, I missed this. But I also don't know which version will this API be. I guess major increase, so I will add 6

Reply:
Hmm catually I deleted this. It will always be up to date as a part of OSD demos.